packages/site/app/budget/ChiffresClefs.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import {useActiveUsers} from 'app/stats/ActiveUsers';
+import {useEvolutionTotalActivation} from 'app/stats/EvolutionTotalActivationParType';
+
+const ChiffresClefs = () => {
+  const {data: evolutionActivation} = useEvolutionTotalActivation('', '');
+  const collectivitesActivees = evolutionActivation
+    ? evolutionActivation.courant.total
+    : undefined;
+
+  const {data: activeUsers} = useActiveUsers('', '');
+  const totalUtilisateurs = activeUsers
+    ? activeUsers.courant.total_utilisateurs
+    : undefined;
+
+  return (
+    <div className="grid sm:max-lg:grid-cols-2 xl:grid-cols-2 md:gap-10">
+      {collectivitesActivees && (
```

Comment:
Quand la variable `collectivitesActivees` vaut `0` (fort heureusement ce n'est pas le cas en prod) cela affiche un beau `0` sur la page alors qu'on voudrait ne rien afficher ici...
C'est pour cela qu'il vaut mieux utiliser systématiquement l'opérateur ternaire ou `!!` pour faire le test sur un booléen plutôt qu'un nombre (voir par exemple [cet article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)).
Je vais préparer une PR séparée pour ajouter une règle de lint car c'est une erreur assez commune....